code/src/components/Start.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { game, fetchStartPosition } from '../reducers/game';
+
+import './start.css';
+
+// Start the game by typing a name and press start
```

Comment:
Great job with comments explaining each section, it makes it much easier for us as reviewers and others to understand 👍